mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -258,12 +274,21 @@ def create_simple_endpoints(self, interfaces, spec):
 
             ep.set_vni(spec['vni'])
             ep.set_vpc(spec['vpc'])
-            ep.set_net(spec['net'])
+            """
+            'subnet' is an optional field for arktos
+            'ip' is also an optional field, and needs to fall within subnet's CIDR
+            since both fields are optional, we need force subnet to have the same
+            CIDR range as vpc,
+            OR arktos needs check wheter the ip and subnet is valid
+            """
+            ep.set_net(net_info.get('subnet', spec['net']))
+            ep.set_ip(net_info.get('ip', ''))
 
             ep.set_mac(interface.address.mac)
             ep.set_veth_name(interface.veth.name)
             ep.set_veth_peer(interface.veth.peer)
             ep.set_droplet(spec['droplet'].name)
+            ep.set_k8s_pod_name(spec['name'])
```

Comment:
Is this a temporal design and will be changed after 9/15?  I think it should be the other way around:  Arkos side tracks the endpoint id in pod nic status.  So that Mizar can be neutral and can support any computing client. With current design it will limit what computing clients Mizar can support. Also it requires more and more such fields when we support more computing clients :)

Reply:
I need remove this..... sorry, I missed this part